### ISSUE TITLE
5 Powerful mental models from Charlie Munger

### ISSUE THREAD

**Issue by lntutor** [lntutor/til#46]
TL;DR

1. Spend less time trying to be brilliant and more time trying to avoid stupidity.

2. Always operate inside your circle of competence, anything that you do outside your circle of competence is gambling.
3. When you are weighing alternative hypotheses the one with the fewest necessary assumptions should be chosen.
4. You have to decide whether you want to have a better today at the cost of tomorrow or a better tomorrow at the cost of today.

5. Always think about the second-order consequences of your actions, that way you will be able to keep a long-term mindset.

https://rattibha.com/thread/1522245339616477184?lang=en
